add.cgi

```diff
@@ -48,12 +48,23 @@ my $usersfile = "$dir/$htusers";
 foreach my $clash ("AuthUserFile", "AuthType", "AuthName") {
 	my $dirclash = &apache::find_directive($clash, $conf);
 	if ($dirclash) {
-		&error(&text('add_eclash3', $file, $clash));
+		&error(&text('add_egclash', $file, $clash));
 		}
 	}
--r $usersfile && &error(&text('add_eclash2', $usersfile));
+-r $usersfile && &error(&text('add_egclash', $usersfile));
 -l $file && &error(&text('add_esymlink', $file));
 
+# Add protected directory in other webserver plugins
+foreach my $p (&virtual_server::list_feature_plugins()) {
+	my $err = &virtual_server::plugin_call($p,
```

Comment:
Maybe add a `plugin_defined` call to check if `feature_add_protected_dir` exists before calling it..

Reply:
Alright, good idea. Fixed on the latest commit!